### ISSUE TITLE
Questions about using TFT

### ISSUE THREAD

**Issue by YUKUN-XIAO** [jdb78/pytorch-forecasting#1361]
Is it possible to use TFT to train multiple time series at the same time and train an independent model for each time series.

**Comment by YUKUN-XIAO** [comment]
Is it possible to train separate models for each time series at the same time?
How can we end up with as many models as the number of different time series？

**Comment by manitadayon** [comment]
Yes, if you are feeding time series one after the other to TFT in a loop, it can find one independent model per time series, but why do you want to do that, since you are treating it like a global model. 
Maybe you are asking if is it possible to develop a model adaptive to each time series: Then you can the group normalizer as the target normalizer.

**Comment by YUKUN-XIAO** [comment]
Thanks. Because we have a requirement, we need to train a separate and effective model for the time series of each ocean sensor. Through research, I found an example of using TFT to predict the number of passengers for each pick-up location, but it seems that only one model was trained in the case. So come and ask. Case URL ：https://github.com/christy/AnyscaleDemos/blob/main/forecasting_demos/Ray_v2/ray_air/pytorch_forecasting.ipynb

**Comment by YUKUN-XIAO** [comment]
Is TFT trained by distinguishing each time series by group_ids? 
Then train each time series, and finally fuse a global model? 
Can I extract the models trained for each group_ids individually?
